server_src/routes/index.js

```diff
@@ -43,6 +43,7 @@ async function welcome() {
       } else {
         res.render("welcome", {
           page: "Welcome",
+          octoFarmPageTitle: process.env.OCTOFARM_SITE_TITLE,
```

Comment:
Try to use the process.env[AppConstants.OCTOFARM_SITE_TITLE_KEY] so we decouple the name from the variable intended

Reply:
Will PR now, aye was a bit but it works and it's in one place.